java/tests/com/google/privacy/differentialprivacy/BoundedMeanTest.java

```diff
@@ -372,6 +387,157 @@ public void computeResult_resultAlwaysInsideProvidedBoundaries() {
     }
   }
 
+  /** Calling computeConfidenceInterval before noised mean is computed throws an exception. */
+  @Test
+  public void computeConfidenceInterval_calledBeforeResult() {
+    assertThrows(IllegalStateException.class, () -> mean.computeConfidenceInterval(ALPHA));
+  }
+
+  @Test
+  public void computeConfidenceInterval_callsNoiseComputeConfidenceIntervalCorrectly() {
+    int maxPartitionsContributed = 1;
+    int maxContributionsPerPartition = 3;
+    double alpha = 0.5;
+    mean =
+        BoundedMean.builder()
+            .epsilon(EPSILON)
+            .delta(DELTA)
+            .noise(noise)
+            .maxPartitionsContributed(maxPartitionsContributed)
+            .maxContributionsPerPartition(maxContributionsPerPartition)
+            .lower(2.0)
+            .upper(10.0)
+            .build();
+
+    mean.addEntry(3.0);
+    mean.addEntry(7.0);
+    mean.computeResult();
+    mean.computeConfidenceInterval(alpha, alpha / 2);
+
+    // Confidence interval for normalized sum.
+    verify(noise)
+        .computeConfidenceInterval(
+            eq(/* x1 + x2 - midpoint * count = 3 + 7 - 6 * 2 = */ -2.0),
+            eq(maxPartitionsContributed),
+            eq(/* maxContributionsPerPartition * (upper - lower) / 2 = 3 * (10 - 2) / 2 = */ 12.0),
+            eq(EPSILON / 2.0),
+            eq(DELTA / 2.0),
+            eq(alpha / 2));
+
+    // Confidence interval for count.
+    verify(noise)
+        .computeConfidenceInterval(
+            eq(/* count */ 2L),
+            eq(maxPartitionsContributed),
+            eq(/* sensitivity of count  = maxContributionsPerPartition */ (long) maxContributionsPerPartition),
```

Comment:
same comments as above, and non need to specify that this is for count, line 427 says that